submissions/MosQuitO404/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Hooli-style-Popup</title>
+</head>
+
+<body>
+    <header class="popup-header">
+        <nav class="nav-items">
+            <ul class="nav-items-ul">
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Home</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">About</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Pricing</a>
+                </li>
+            </ul>
+            <input type="checkbox" id="central-button">
+            <label id="central-button-label" for="central-button"><img src="sources/icons/free-icon-menu-7512756.png" id="central-button-logo" alt=""></label>
+            <ul class="nav-items-ul">
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Blog</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Integrations</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="https://github.com/MosQuitO404" target="_blank">Contacts</a>
+                </li>
+            </ul>
+            <div id="popup-section">
```

Comment:
When a user opens the popup focus must go to the first interactive element, usually, it's a close button, but in our case when a user opens the popup and presses the` Tab` key focus must go to the first link inside the popup. In your case order of focused elements is broken. :cry:

Reply:
Hi! Bugs were fixed and added adaptive design. Review pls.